### PR TITLE
Setting up Python Course Outline folder

### PR DESCRIPTION
**Pull request by ritikramuka**

### CODE REVIEW

File: Python/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* Code Indentation in Python
+* Why do we need functions?
+* Implementing basic functions
+## Fundamentals Of Python
+* What are Arrays?
+* Implementing Arrays in Python
+* Lists
+    * What are Lists in Python
+    * Implementing Lists in Python
+    * Basic operations on Lists * 2
```
Comment (aman5898) [inline Python/README.md]: Please define

File: Python/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* Tuples
+    * What are Tuples in Python
+    * Implementing Tuples in Python
+    * Basic operations on Tuples * 2
```
Comment (aman5898) [inline Python/README.md]: Please define

File: Python/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+* Sets
+    * What are Sets in Python
+    * Implementing Sets in Python
+    * Union in Sets
+    * Basics operations on Sets * 2
+    * Updates in Sets Data Structure in new version of python
```
Comment (aman5898) [inline Python/README.md]: Define More function of Sets

File: Python/README.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    * Concatenation of String
+        * What happens when concat string and number
+        * Format function
+* Advanced Functions
+    * Lambda Functions
+    * Basics functions containing lambda functions
+* Dictionary
+    * What is a dictionary?
+    * Implementing a dictionary in Python.
+    * Functions of dictionary
```
Comment (aman5898) [inline Python/README.md]: Define More functions of Dictionary

File: Python/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* Racap of List, Sets, Tuples and Dictionary
```
Comment (aman5898) [inline Python/README.md]:
```suggestion
* Recap of List, Sets, Tuples and Dictionary
```

**Review by aman5898 — CHANGES_REQUESTED** [review]
Added Some Comments, Please check

File: Python/README.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+* What are Arrays?
+* Implementing Arrays in Python
+* Lists
+  * What are Lists in Python
+  * Implementing Lists in Python
+  * Basic operations on Lists * 2
+* Tuples
+  * What are Tuples in Python
+  * Implementing Tuples in Python
+  * Basic operations on Tuples * 2
```
Comment (aman5898) [inline Python/README.md:47]: Name those operations here

File: Python/README.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+* Sets
+  * What are Sets in Python
+  * Implementing Sets in Python
+  * Union in Sets
+  * Basics operations on Sets * 2
```
Comment (aman5898) [inline Python/README.md:52]: Name those operations

File: Python/README.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+  * Concatenation of String
+    * What happens when concat string and number
+    * Format function
+* Advanced Functions
+  * Lambda Functions
+  * Basics functions containing lambda functions
+* Dictionary
+  * What is a dictionary?
+  * Implementing a dictionary in Python.
+  * Functions of dictionary
```
Comment (aman5898) [inline Python/README.md:85]: Name those functions

**Review by aman5898 — CHANGES_REQUESTED** [review]

File: Python/README.md (hunk 6 above)
Comment (aman5898) [inline Python/README.md]:
```suggestion
  * Basic operations on Tuples * 2
```
Name those operaitons

File: Python/README.md (hunk 7 above)
Comment (aman5898) [inline Python/README.md]: Name operations

File: Python/README.md (hunk 1 above)
Comment (ritikramuka) [reply Python/README.md]: done!

File: Python/README.md (hunk 2 above)
Comment (ritikramuka) [reply Python/README.md]: done!

File: Python/README.md (hunk 3 above)
Comment (ritikramuka) [reply Python/README.md]: done

File: Python/README.md (hunk 4 above)
Comment (ritikramuka) [reply Python/README.md]: done

File: Python/README.md (hunk 8 above)
Comment (ritikramuka) [reply Python/README.md:85]: addressed before!

File: Python/README.md (hunk 6 above)
Comment (ritikramuka) [reply Python/README.md]: addressed before!

File: Python/README.md (hunk 7 above)
Comment (ritikramuka) [reply Python/README.md]: Addressed before!

**Pull request closed without merging**